python/nano/src/bigdl/nano/pytorch/onnx/onnxrt_inference.py

```diff
@@ -184,6 +188,66 @@ def inference(self,
 # on_fit_start (LightningModule method overwrite)
 def on_fit_start(self):
     self._ortsess_up_to_date = False
+    self.exit_onnx()
+
+
+def train(self, mode=True):
+    self.exit_onnx()
+    self._train_mode(mode)
```

Comment:
why add a new `_train_mode()` instead of calling `super().train()`?

Reply:
`super()` need a class name when it is defined outside a class. And I can't get it here.

Reply:
`self._ortsess_up_to_date = False`?